test/general/version.pass.cpp

```diff
@@ -15,23 +15,21 @@
 
 #include "support/test_config.h"
 
-#include _PSTL_TEST_HEADER(execution)
-
-#include <oneapi/dpl/pstl/onedpl_config.h>
-
-#include "support/utils.h"
+#include _PSTL_TEST_HEADER(version)
 
 #if !__has_include(<execution>)
-static_assert(_PSTL_VERSION == 11000);
-static_assert(_PSTL_VERSION_MAJOR == 11);
+static_assert(_PSTL_VERSION == 14000);
+static_assert(_PSTL_VERSION_MAJOR == 14);
 static_assert(_PSTL_VERSION_MINOR == 00);
 static_assert(_PSTL_VERSION_PATCH == 0);
 #endif
 
 static_assert(ONEDPL_VERSION_MAJOR == 2022);
-static_assert(ONEDPL_VERSION_MINOR == 6);
+static_assert(ONEDPL_VERSION_MINOR == 7);
 static_assert(ONEDPL_VERSION_PATCH == 0);
 
+#include "support/utils.h"
```

Comment:
What's the point to move it below `static_assert`s? Is it because we want to make sure that we `<version>` header has everything we need?

Reply:
Exactly.
It is actually a good practice when writing tests to include any utility headers as late as possible, to minimize chances of artificially introducing extra code (incl. standard headers) that might interfere with the tested API.